cli/command/formatter/sorter.go

```diff
@@ -0,0 +1,319 @@
+package formatter
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"time"
+)
+
+const (
+	ascendantKey  = "asc"
+	descendantKey = "dsc"
```

Comment:
I wonder `desc` is widely used abbreviation.

Google result:
- "sort asc dsc": 350,000  hits
- "sort asc desc": 15,100,000 hits